client/shipping-label/state/test/reducer.js

```diff
@@ -123,6 +126,41 @@ describe( 'Label purchase form reducer', () => {
 		expect( state.form.rates.available ).to.eql( {} );
 	} );
 
+	it( 'ADD_ITEMS moves a set of items from their original packaging to selected package', () => {
+		let existingState = hoek.clone( initialState );
+		existingState.form.packages.selected.weight_2_custom1 = {
+			items: [
+				{
+					product_id: 789,
+					weight: 4.5,
+				},
+			],
+		};
+		existingState.form.packages.selected.weight_1_custom1.items[ 1 ] = {
+			product_id: 457,
+			weight: 3.4,
+		};
+
+		existingState = [
+			setAddedItem( 'weight_0_custom1', 0, true ),
+			setAddedItem( 'weight_1_custom1', 0, true ),
+			setAddedItem( 'weight_1_custom1', 1, true ),
+		].reduce( reducer, existingState );
```

Comment:
Why modify initial state for package data, but then use the reducer for selecting items to add?

My gut says that we should either add some expectations around the result of that reducer, or use an initial state that already contains the necessary value for `addedItems`.